Content.Server/Silicons/Borgs/BorgSystem.cs

```diff
@@ -242,6 +244,12 @@ private void OnBrainMindAdded(EntityUid uid, BorgBrainComponent component, MindA
         _mind.TransferTo(mindId, containerEnt, mind: mind);
     }
 
+    private void OnPointAttempt(EntityUid uid, BorgBrainComponent component, PointAttemptEvent args)
+    {
+        if (!HasComp<BorgChassisComponent>(uid))
```

Comment:
99% sure this wont work

Reply:
I just tested it. PB couldn't point until I put it inside a chassis. Lost the ability when I took it out again.

Do we expect it to break anything else?

Reply:
the posibrain isnt doing the pointing the borg is

Reply:
![05:57:36](https://github.com/space-wizards/space-station-14/assets/39013340/4a38fb3a-6fdb-49bc-8f5d-2bc128a350a6)

when in the chassis the minds OwnedEntity is the borg chassis, not the brain
BorgChassis is never added to the brain

shrimply just have borgbrain always cancel it like meat brain

Reply:
Ohhh I see, the check isn't needed then.